docs/FAQ.md

```diff
@@ -0,0 +1,52 @@
+### Frequently Asked Questions
+
+#### Table of Contents
+* [What is KUDO?](#what-is-kudo)
+* [When would you use KUDO?](#when-would-you-use-kudo)
+* [What is a Framework](#what-is-a-framework)
+* [What is a Frameworkversion?](#what-is-a-frameworkversion)
+* [What is an Instance?](#what-is-an-instance)
+* [What is a Planexecution?](#what-is-a-planexecution)
+* [What is a Parameter?](#what-is-a-parameter)
+* [What is a Trigger?](#what-is-a-trigger)
+* [When I create a Framework, it is automatically creating new CRDs?](#when-i-create-a-framework-it-is-automatically-creating-new-crds)
+* [How does it work from a RBAC perspective?](#how-does-it-work-from-a-rbac-perspective)
+* [Is the dependency model an individual controller per workload?](#is-the-dependency-model-an-individual-controller-per-workload)
```

Comment:
We should add FAQs comparing it to other projects in the space, but not needed in this PR (but a motivation to have groups)